src/test_payload_json_utils.cc

```diff
@@ -35,47 +35,6 @@
 
 namespace triton::perfanalyzer {
 
-TEST_CASE("PayloadJsonUtils::GetSession")
```

Comment:
Again, this functionality was moved into the PayloadDatasetManager.

Session ID no longer is stored inside the payload JSON, so there's no need to have functionality (or a test) for `PayloadJsonUtils` for fetching the session ID.

Session ID is now part of the input data JSON top-level inference inputs themselves, and is fetched via the `PayloadDatasetManager` (`DataLoader` under the hood).